registration/forms.py

```diff
@@ -155,6 +164,23 @@ def __init__(self, *args, **kwargs):
                     if name not in self.can_be_modified:
                         field.widget.attrs['readonly'] = True
 
+    def _check_password_strength(self, password_strength):
+        """
+        Check that client submitted password strength, and that value is in expected range.
+
+        Return password_strength as-is if there are no issues, otherwise return None.
+        """
+        if password_strength is None:
+            logger.warning('Did not receive password strength from client.')
+        elif password_strength not in range(5):
+            logger.warning(
+                'Received suspicious value for password strength. '
+                'Value should be in range [0, 4]. '
+                'Observed value: {password_strength}'.format(password_strength=password_strength)
```

Comment:
Pylint is complaining about the use of `.format` here. It says you should use % formatting instead.

This might seem weird but it can be useful for grouping error messages, when using something like Sentry for example.